src/components/screen/AdminScreen.jsx

```diff
@@ -28,8 +29,8 @@ const AdminScreen = () => {
   }, [dispatch]);
 
   useEffect(() => {
-    dispatch(fetchReleases());
```

Comment:
Treng vi ikke å hent inn releases i AdminScreen?

Reply:
Der sto fetchReleases to plassa, e fjerna berre et duplikat

Reply:
Den e grei